### DISCUSSION TITLE
Chromecast with Google TV support is required

### DISCUSSION THREAD

**Discussion by fghj69119** [AppGround-io/bluetooth-keyboard-and-mouse-support#229] [Q&A] [answered]
It doesn't seem to work on Chromecast with Google TV. 
Is it possible to realize it later?

**Comment by AdamKane41** [comment]
Just imagine the infinite power that bLeK could unleash on a chromecast with google tv device!
Gotta be patient teeeeheeee! :)
Peace

**Comment by parkerlreed** [comment, marked as answer]
It works fine but the default Bluetooth pairing won't find it as it filters devices. There's tons of applications on the store that you can sideload to get this working.

https://play.google.com/store/apps/details?id=com.waylonhuang.bluetoothpair&hl=en_US&gl=US

**Comment by AdamKane41** [comment]
Got it working using blutooth Mac  Adress method....and it is as expected; DELICIOUS! THANKS! peace

**Comment by AdamKane41** [comment]
BleK works with GOOGLE CHROMECAST PLUS GOOGLE TV. This issues should be closed, peace

**Comment by AdamKane41** [comment]
After update to Android 12, Can't "CLICK" OR  "SWIPE" ... This happends with EVERY other remote apps out there...

**Comment by richardelima** [comment]
I can see and move the mouse, but cannot click on Android tv

**Reply by AdamKane41** [reply]
Hehehe once again it now allows clicking and every other possible keys. it's been 3 times now that this beaviour went and gone... Seem to be related to Google Chromecast with Google TV's various systems updates. Peace 🐛
